qutebrowser/browser/commands.py

```diff
@@ -1847,13 +1663,23 @@ def debug_webaction(self, action, count=1):
             action: The action to execute, e.g. MoveToNextChar.
             count: How many times to repeat the action.
         """
-        member = getattr(QWebPage, action, None)
-        if not isinstance(member, QWebPage.WebAction):
+        tab = self._current_widget()
+
+        if tab.backend == tabmod.Backend.QtWebKit:
```

Comment:
Right - what I had in mind is "let's avoid directly accessing `_widget` whenever possible" and "nothing else will use this, so let's not make a full abstraction". But I agree now, better to just access `_widget` in `:debug-webaction` for this case.

I'm still open to have a more fine-grained webaction API, but I think this should be added after finding out something would actually need it :wink: